src/main/java/duke/util/Ui.java

```diff
@@ -0,0 +1,108 @@
+package duke.util;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Handles the print messages for user interaction.
+ */
+public class Ui {
+
+    private Scanner sc = new Scanner(System.in);
+    private String line = "    ____________________________________________________________";
+
+    /**
+     * Shows the welcome message.
+     */
+    public void showWelcome() {
+        System.out.println(line + "\n"
+                + "     Hello! I'm Duke\n"
+                + "     What can I do for you?\n" + line);
+    }
+
+    /**
+     * Shows the error message for loading errors.
+     * @param e error message when caught
+     */
+    public void showLoadingError(String e) {
+        System.err.println(e);
+    }
+
+    /**
+     * Shows a general error message.
+     * @param e error message when caught
+     */
+    public void showError(String e) {
+        System.err.println(e);
+    }
+
+    /**
+     * Reads the next line of user input.
+     * @return a string representation of user input
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Displays the line breaks in between commands.
+     */
+    public void showLine() {
+        System.out.println(line);
+    }
+
+    /**
+     * Shows the message when user deletes a task.
+     * @param task deleted task object
+     * @param numOfTasks number of tasks left on the list
+     */
+    void showDeleted(Task task, int numOfTasks) {
```

Comment:
On one hand, putting all the print statements inside the `Ui` class makes it so only one class has to be changed when dealing with UI issues. On the other hand, this tightly couples the `Ui` class with the `Command` classes, making them highly dependent on this class. Just some food for thought, for me as well!